apps/frontend/src/app/App.scss

```diff
@@ -51,8 +53,8 @@ body {
     $random-x: random(1000000) * 0.0001vw;
     $random-offset: random_range(-100000, 100000) * 0.0001vw;
     $random-x-end: $random-x + $random-offset;
-    $random-x-end-yoyo: $random-x + ($random-offset / 2);
```

Comment:
can you explain why this was done?

Reply:
This was throwing an error for some reason, division isnt supported in sass like this